config/env_development.json

```diff
@@ -5,5 +5,6 @@
     "protocol": "https",
     "domain": "dev03.chatgrape.com",
     "path": "chat"
-  }
+  },
+  "startInBackgroundWhenAutostarted": true
```

Comment:
do we need this option at all? Is there a case where you would start it not in background when autostarted and create a separate build with that flag turned off?

Reply:
it can be set in `.graperc` anyway. I was not sure if I need in the env jsons too. can I remove it?